src/components/PostListing/PostListing.css

```diff
@@ -0,0 +1,3 @@
+.word-break {
+  hyphens: auto;
```

Comment:
CSS vendor prefixes can help us provide standardized behavior cross browsers.

```suggestion
  -webkit-hyphens: auto;
  -moz-hyphens: auto;
  -ms-hyphens: auto;
  hyphens: auto;
```

Reply:
I was confused to add this or not like  I checked on Mozilla and chrome I works fine